src/aerialmap_display.cpp

```diff
@@ -462,69 +468,43 @@ void AerialMapDisplay::assembleScene()
   tileCache_.purge({ tileId, blocks_ });
 }
 
-void AerialMapDisplay::transformAerialMap()
+void AerialMapDisplay::transformTileToMapFrame()
```

Comment:
I'd leave it as is. Whats the AerialMap exactly? the tiles's scene objects, including geometry and texture? since this term has no member variable or clear definition, its a bit shallow to use.